schema/provider-registry.json

```
@@ -11,10 +11,16 @@
       "description": "Whether to use or not the secrets manager. If not, secrets will be stored in the browser (local storage)",
       "default": true
     },
-    "AIprovider": {
+    "UniqueProvider": {
+      "type": "boolean",
+      "title": "One provider for chat and completer",
```

Maybe something like the following could also work?

```suggestion
      "title": "Use the same provider for chat and completer",
```